docs/models/advanced/modeldata.md

```diff
@@ -0,0 +1,61 @@
+Model Data
+==========
+
+Model data is the successor of "extended blockstates" allowing modders to pass in extra data to be used in [`IBakedModel#getQuads`][getquads] when rendering blocks. This is particularly useful in complex models such as for camouflage blocks which use other block's models as an example. This is not to be confused with a [`TileEntityRenderer`][ter] which is used for dynamic rendering such as for animated blocks like a chest or any blocks which cannot be defined by a list of `BakedQuad`s.
+
+IBakedModels vs TERs
+------------------
+
+An [`IBakedModel`][bakedmodel] simply provides a list of `BakedQuad`s to the `BlockModelRenderer` which are then drawn to the screen. There are other properties available which define aspects which effect rendering such as whether the model supports ambient occlusion (a rendering effect). Most [`IBakedModel`][bakedmodel]s come from when a model file is loaded and then "baked" so that the model does not need to be loaded again.
+
+A [`TileEntityRenderer`][ter] or [`TER`][ter] (previously [`TileEntitySpecialRenderer`] or [`TESR`]) is used to render blocks in a way that cannot be represented with a static baked model (JSON, OBJ, B3D, others).
```

Comment:
Why are the old names in brackets?

Reply:
I copied the line from the TER documentation here as I felt that explained a TER better than I could have. I'll remove the reference